python/tank/bootstrap/constants.py

```diff
@@ -19,7 +19,7 @@
 # if major changes happen to the way cloud based configs are handled
 # by the system, for example requiring any existing deployed cloud
 # configs to be re-deployed, this version number should be incremented.
-BOOTSTRAP_LOGIC_GENERATION = 5
+BOOTSTRAP_LOGIC_GENERATION = 6
```

Comment:
We're bumping this id so that existing cached configurations are rewritten with the proper `pipeline_configuration.yml` values.